int64 a
int64 b

float64 joint_1
float64 joint_2
float64 joint_3
float64 joint_4

---

int64 sum
int64 master
int64 slave

float64 joint_1
float64 joint_2
float64 joint_3
float64 joint_4

int64 slave_mode_
